modules/openapi-generator/src/test/resources/3_0/rust/petstore.yaml

```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.0
 servers:
-  - url: 'http://petstore.swagger.io/v2'
+  - url: 'https://petstore.swagger.io/v2'
```

Comment:
please revert the change.

the petstore server should be run locally per https://github.com/OpenAPITools/openapi-generator/wiki/Integration-Tests#how-to-add-integration-tests-for-new-petstore-samples